tempodb/encoding/vparquet4/rebatch_trace_test.go

```diff
@@ -0,0 +1,208 @@
+package vparquet4
+
+import (
+	"crypto/rand"
```

Comment:
nit: math/rand good enough?

Reply:
The `Read()` function from `math/rand` is deprecated. There are two alternatives:
* Use `Read()` from `crypto/math`
* Create a `ChaCha8` random number generator from `math/rand/v2` and use it's `Read()` method

I think it would be great to migrate from `math/rand` to `math/rand/v2` but it might be better to do this package wide, or even project wide. There are subtle differences between `math/rand` and `math/rand/v2` and I'm not sure if it creates confusion to use both at the same time.